package_nso_to_oc/xe/xe_network_instances.py

```diff
@@ -213,6 +213,33 @@ def process_rd_rt(temp_vrf, vrf, vrf_index, config_leftover):
 
         del config_leftover["tailf-ned-cisco-ios:vrf"]["definition"][vrf_index]["rd"]
 
+        # IPv4 RT import and export policies
+        temp_policies = {
+            "openconfig-network-instance:inter-instance-policies": {
+                "openconfig-network-instance:apply-policy": {
+                    "openconfig-network-instance:config": {
+                        "openconfig-network-instance:export-policy": [],
+                        "openconfig-network-instance:import-policy": []}}}}
+        if vrf.get("address-family", {}).get("ipv4", {}).get("import", {}).get("ipv4", {}).get("unicast", {}).get(
+                "map"):
+            temp_policies["openconfig-network-instance:inter-instance-policies"][
+                "openconfig-network-instance:apply-policy"]["openconfig-network-instance:config"][
+                "openconfig-network-instance:import-policy"].append(
+                vrf.get("address-family", {}).get("ipv4", {}).get("import", {}).get("ipv4", {}).get("unicast", {}).get(
+                    "map"))
+            del config_leftover["tailf-ned-cisco-ios:vrf"]["definition"][vrf_index]["address-family"]["ipv4"]["import"]
+        if vrf.get("address-family", {}).get("ipv4", {}).get("export", {}).get("map"):
+            temp_policies["openconfig-network-instance:inter-instance-policies"][
+                "openconfig-network-instance:apply-policy"]["openconfig-network-instance:config"][
+                "openconfig-network-instance:export-policy"].append(
+                vrf.get("address-family", {}).get("ipv4", {}).get("export", {}).get("map"))
+            del config_leftover["tailf-ned-cisco-ios:vrf"]["definition"][vrf_index]["address-family"]["ipv4"]["export"]
+        if "ipv4" in vrf.get("address-family", {}) and len(vrf.get("address-family", {}).get("ipv4", {"1": "1"})) == 0:
+            del config_leftover["tailf-ned-cisco-ios:vrf"]["definition"][vrf_index]["address-family"]["ipv4"]
+        temp_vrf.update(temp_policies)
+        print(temp_vrf)
```

Comment:
Just curious if you meant to leave this print statement

Reply:
I'll remove the print statement. Thanks!